<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalendarNotifications</class>
 <widget class="QMainWindow" name="CalendarNotifications">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalendarNotifications</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #191F32;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="alertPushButton">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>311</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border:0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="wardenfort.qrc">
      <normaloff>:/wardenfort/downButton.png</normaloff>:/wardenfort/downButton.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="alertLessButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>307</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border:0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="wardenfort.qrc">
      <normaloff>:/wardenfort/lessButton.png</normaloff>:/wardenfort/lessButton.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="frame">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>89</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="wardenfort.qrc">:/wardenfort/frame.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="triReversedButton">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>280</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border:0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="wardenfort.qrc">
      <normaloff>:/wardenfort/triReversedButton.png</normaloff>:/wardenfort/triReversedButton.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="reportPushButton">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>360</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border:0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="wardenfort.qrc">
      <normaloff>:/wardenfort/downButton.png</normaloff>:/wardenfort/downButton.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="reportLessButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>356</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border:0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="wardenfort.qrc">
      <normaloff>:/wardenfort/lessButton.png</normaloff>:/wardenfort/lessButton.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="calPushButton">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>407</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border:0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="wardenfort.qrc">
      <normaloff>:/wardenfort/downButton.png</normaloff>:/wardenfort/downButton.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="welcome_text">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>50</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    qproperty-alignment: AlignCenter;
	font: 700 10pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
}</string>
    </property>
    <property name="text">
     <string>Welcome, User!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="profilePushButton">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>261</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border:0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="wardenfort.qrc">
      <normaloff>:/wardenfort/downButton.png</normaloff>:/wardenfort/downButton.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="triButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border:0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="wardenfort.qrc">
      <normaloff>:/wardenfort/triButton.png</normaloff>:/wardenfort/triButton.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="profileLessButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>257</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border:0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="wardenfort.qrc">
      <normaloff>:/wardenfort/lessButton.png</normaloff>:/wardenfort/lessButton.png</iconset>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>201</width>
      <height>531</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
    background-color: rgb(44, 60, 75);
    border-top-left-radius: 0px;
    border-bottom-right-radius: 20px;
    border-top-right-radius: 20px;
    border-bottom-left-radius: 0px;
    padding: 30px;
    padding-left: 15px;
    padding-top: 250px;
    font: 700 11pt &quot;Segoe UI&quot;;
    color: rgb(255, 255, 255);
}
QListWidget::item {
    margin-bottom: 25px; /* Adjust the spacing between items by setting a bottom margin */
    border-bottom: 1px solid rgb(52, 78, 102);
}


QListWidget::item:hover {
    background-color:rgb(25, 31, 50);
    border-bottom: 1px solid rgba(255, 255, 255, 0.5); /* Adjust border on hover */
}


QListWidget::item:selected {
    background-color: transparent; /* Make selection color transparent */
    border: none;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Profile</string>
     </property>
     <property name="icon">
      <iconset resource="wardenfort.qrc">
       <normaloff>:/wardenfort/user_icon.png</normaloff>:/wardenfort/user_icon.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Alerts</string>
     </property>
     <property name="icon">
      <iconset resource="wardenfort.qrc">
       <normaloff>:/wardenfort/notif_icon.png</normaloff>:/wardenfort/notif_icon.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Reports</string>
     </property>
     <property name="icon">
      <iconset resource="wardenfort.qrc">
       <normaloff>:/wardenfort/report_icon.png</normaloff>:/wardenfort/report_icon.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Event Calendar</string>
     </property>
     <property name="icon">
      <iconset resource="wardenfort.qrc">
       <normaloff>:/wardenfort/cal_icon.png</normaloff>:/wardenfort/cal_icon.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fort Chat</string>
     </property>
     <property name="icon">
      <iconset resource="wardenfort.qrc">
       <normaloff>:/wardenfort/mess_icon.png</normaloff>:/wardenfort/mess_icon.png</iconset>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="wardenfort">
    <property name="geometry">
     <rect>
      <x>-11</x>
      <y>30</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="wardenfort.qrc">:/wardenfort/wardenfort_word (1).png</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>811</width>
      <height>491</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Inter</family>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
color: white;</string>
     </property>
     <property name="text">
      <string>CALENDAR NOTIFICATIONS</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>821</width>
       <height>491</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="wardenfort.qrc">:/wardenfort/bg.png</pixmap>
     </property>
    </widget>
    <widget class="QListWidget" name="notificationList">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>761</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>notificationList</zorder>
   </widget>
   <zorder>widget</zorder>
   <zorder>listWidget</zorder>
   <zorder>alertPushButton</zorder>
   <zorder>alertLessButton</zorder>
   <zorder>frame</zorder>
   <zorder>triReversedButton</zorder>
   <zorder>reportPushButton</zorder>
   <zorder>reportLessButton</zorder>
   <zorder>calPushButton</zorder>
   <zorder>welcome_text</zorder>
   <zorder>profilePushButton</zorder>
   <zorder>triButton</zorder>
   <zorder>profileLessButton</zorder>
   <zorder>wardenfort</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1088</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="wardenfort.qrc"/>
 </resources>
 <connections/>
</ui>
